.github/workflows/build-test-measure.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
       - develop
       # Include all release branches.
-      - '[0-9]+.[0-9]+'
+      - "[0-9]+.[0-9]+"
```

Comment:
Use of double quotes instead of single quotes is just to improve convention?

Reply:
Formatter did this. Will revert such changes.